src/pages/home/home.ts

```diff
@@ -78,7 +78,7 @@ export class HomePage {
       app: 'bitpay',
       linkText: 'View Directory',
       link: 'https://bitpay.com/directory/?hideGiftCards=true',
-      imgSrc: 'assets/img/gumball-3.svg',
+      imgSrc: 'assets/img/icon-merch-dir.svg',
```

Comment:
Missing icon. Probably it's a typo and you wanted use this new one: `mcc-icons/merch-dir.svg`